File: EquilibriumIndexTest.php
<?php
require_once("./EquilibriumIndex.php");
class EquilibriumIndex extends PHPUnit_Framework_TestCase
{
	public function testCase()
	{
		// Should be 3
		$arr = [1,1,1,1,2,2];
		$this->assertEquals(3, getEquilibriums($arr));
		// Should be 3
		$arr = [1.0,1,1,1,2,2];
		$this->assertEquals(3, getEquilibriums($arr));
		// Should be -1 because no index 
		$arr = [1,1,1,1,1,2,2];
		$this->assertEquals(-1, getEquilibriums($arr));
	}
}

File: EquilibriumIndexTest.py
from EquilibriumIndex import eqindex

d = ([-7, 1, 5, 2, -4, 3, 0],
	[2, 4, 6],
	[2, 9, 2],
	[1, -1, 1, -1, 1, -1, 1])
 
for data in d:
	print("Original Array: %r" % data)
	print("Equilibrium Index: %r" % eqindex(data))
